packages/@aws-cdk/aws-dynamodb/test/integ.dynamodb.ondemand.ts

```diff
@@ -49,7 +49,7 @@ new Table(stack, TABLE, {
 
 const tableWithGlobalAndLocalSecondaryIndex = new Table(stack, TABLE_WITH_GLOBAL_AND_LOCAL_SECONDARY_INDEX, {
   pointInTimeRecovery: true,
-  serverSideEncryption: true,
+  encryption: TableEncryption.AWS_MANAGED,
```

Comment:
I feel it would be preferable to add a new integration test rather than update the existing one. It then ensures we're not breaking the old behavior, too.